src/const.js

```diff
@@ -1,3 +1,9 @@
+import {
```

Comment:
I'd love to find a JSCS rule for this, but for now I'll comment here: our import style for lines longer than one line is:

```js
import { NO_DOCUMENT_WRITE, NSI_DNS_SERVICE_RESOLVE,
          NSI_SOUND_PLAY } from 'messages';
```